src/utils/atomicSwap.js

```diff
@@ -340,3 +333,78 @@ export const updatePersistentStorage = (proposalList) => {
 
     walletUtil.setListenedProposals(simplifiedStorage);
 }
+
+/**
+ * Calculates the object that will be stored in Redux, containing all the helper values for a more
+ * enriched exhibition on the Atomic Swap screens
+ * @param {string} proposalId
+ * @param {string} password
+ * @param {string} partialTx
+ * @param {HathorWallet} wallet
+ * @param [options]
+ * @param {boolean} [options.signatures] Optional signatures data for the proposal
```

Comment:
This is a string, right?

Reply:
You're right! Fixed on [f2b09c4](https://github.com/HathorNetwork/hathor-wallet/pull/400/commits/f2b09c49a2fbc9427b8adffcd7652d8d226269e4)